content/projects/oop/animals/part2/_index.md

```diff
@@ -156,6 +170,25 @@ Test -> Does cat eat food should Fail
 
 ```
 
+
+### For Python
+
+Use pytest to test your code.
+
+Please refer to the following to find out more: {{% contentlink path="topics/python-specific/automated-testing-with-pytest" %}}
+
+```
+# Python
+
+dog = Dog()
+dog.eats()    # returns 'Food'
+dog.sounds()  # returns 'Bark'
+
+```
+
+Remember the correct naming convention for the tests in your `test_animals.py` file.
```

Comment:
I think it would be better to remind all the learners to remember their naming file, not just the python ones. like make a general reminder, otherwise then remind all learners under each language section

Reply:
If there is a general rule that works for all then for sure that would be awesome, but I'm a bit clueless on the other languages testing process and if they differ from python 🙈
Do you think it can be made general?